packages/services/src/contents/index.ts

```diff
@@ -1080,6 +1115,14 @@ export class Drive implements Contents.IDrive {
     this._apiEndpoint = options.apiEndpoint ?? SERVICE_DRIVE_URL;
     this.serverSettings =
       options.serverSettings ?? ServerConnection.makeSettings();
+    const contentProviderRegistry = new ContentProviderRegistry();
+    const restContentProvider = new RestContentProvider({
+      apiEndpoint: this._apiEndpoint,
+      serverSettings: this.serverSettings,
+      fileChanged: this.fileChanged
+    });
+    contentProviderRegistry.register(restContentProvider);
+    this.contentProviderRegistry = contentProviderRegistry;
```

Comment:
See the other suggestion on moving instantiation of `ContentProviderRegistry` to the member definition (and making it readonly).

```suggestion
    const restContentProvider = new RestContentProvider({
      apiEndpoint: this._apiEndpoint,
      serverSettings: this.serverSettings,
      fileChanged: this.fileChanged
    });
    this.contentProviderRegistry.register(restContentProvider);
```